CriteoMoPubAdapter/Sources/CriteoMoPubAdapter/CRNativeAdAdapter.m

```diff
@@ -57,6 +57,10 @@ - (NSURL *)defaultActionURL {
   return nil;
 }
 
+- (BOOL)enableThirdPartyClickTracking {
```

Comment:
Are you sure we want to do this? vs letting MoPub handling clicks with maybe passing some click event URL?

Reply:
Handling clicks ourselves would give us better control, i.e. opening in the default browser. MoPub SDK is known to open clicks in a webview browser so we want to avoid that (applying MoPub click scheme to overcome this issue, like on App Bidding integrations, is not tested for Native ads).

Reply:
Ok, was asking as this control comes with responsibilities ;)